.changeset/little-pillows-fry.md

```diff
@@ -0,0 +1,15 @@
+---
+"@salt-ds/core": minor
+---
+
+`Text` component has two new variants: `TextAction` and `TextNotation`. See [Storybook](https://storybook.saltdesignsystem.com/?path=/story/core-text-text-qa--all-variants-grid) for more information.
```

Comment:
would it be better to link to the site? also wondering if this links like this change in the future how that would impact changelogs

Reply:
good shout - will update link + add them to site's Text examples